gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/DatastoreTest.java

```diff
@@ -691,6 +696,37 @@ public void testQueryPaginationWithLimit() throws DatastoreException {
   }
 
   @Test
+  public void testEventuallyConsistentQuery() {
+    DatastoreRpcFactory rpcFactoryMock = EasyMock.createStrictMock(DatastoreRpcFactory.class);
+    DatastoreRpc rpcMock = EasyMock.createStrictMock(DatastoreRpc.class);
+    EasyMock.expect(rpcFactoryMock.create(EasyMock.anyObject(DatastoreOptions.class)))
+        .andReturn(rpcMock);
+    com.google.datastore.v1beta3.ReadOptions readOption =
+        com.google.datastore.v1beta3.ReadOptions.newBuilder()
+            .setReadConsistencyValue(ReadConsistency.EVENTUAL_VALUE)
+            .build();
+    com.google.datastore.v1beta3.GqlQuery query =
+        com.google.datastore.v1beta3.GqlQuery.newBuilder()
+            .setQueryString("FROM * SELECT *")
+            .build();
+    com.google.datastore.v1beta3.RunQueryRequest.Builder expectedRequest =
+        com.google.datastore.v1beta3.RunQueryRequest.newBuilder()
+            .setReadOptions(readOption)
+            .setGqlQuery(query)
+            .setPartitionId(PartitionId.newBuilder().setProjectId(PROJECT_ID).build());
+    EasyMock.expect(rpcMock.runQuery(expectedRequest.build()))
+        .andReturn(RunQueryResponse.newBuilder().build());
+    EasyMock.replay(rpcFactoryMock, rpcMock);
+    Datastore mockDatastore = options.toBuilder()
```

Comment:
Nit: I wouldn't call this `mockDatastore` as the mock suffix/prefix reminds me of an EasyMock mock which this datastore instance is not.